Topics/Software_Engineering/Scrum Planning Meeting.md

```diff
@@ -0,0 +1,32 @@
+# Scrum Planning Meeting
+
+## Introduction
+In the Scrum framework, sprints function as short, consistent cycles during which a set amount of work is completed. Since they are the heartbeat of the framework, ensuring sprints are well planned, or ensuring a successful Sprint Planning Meeting, is a crucial part of the framework.
```

Comment:
Or, if your main focus is on Sprint planning, which I believe is one of the steps of Scrum planning (and not equivalent to it), maybe your title should be "Sprint Planning Meeting" instead, and this file can be added under the more general topic of Scrum Planning.

Reply:
> Or, if your main focus is on Sprint planning, which I believe is one of the steps of Scrum planning (and not equivalent to it), maybe your title should be "Sprint Planning Meeting" instead, and this file can be added under the more general topic of Scrum Planning.

I agree! This was a typo on my end - thank you for catching that!

Reply:
> > It would be helpful to start off with defining the Scrum framework first and what it's used for. This will give the reader a clearer understanding of what this document is about and how they can use it.
> 
> From the _A2-Contribution Plan_, I believe another student has chosen to give an "explanation on what the Scrum Framework is" and since their topic would both, introduce Scrum and touch upon Sprint Planning, I think defining the Scrum Framework here would be redundant.
> 
> Based on this organization, I assume that student's entry would be linked in _1. Scrum_ and my topic would be linked underneath theirs, making Sprint Planning a subtopic of Scrum. ![image](https://user-images.githubusercontent.com/95773832/284648847-7d3ccd81-5151-4a23-9349-88901dd8dbe9.png)

Alright, that makes sense! If your main topic is Sprint Planning, then the Scrum topic should probably covered by another students, as you mentioned. If you don't discuss Scrum Planning specifically and the definition isn't very relevant to the purpose of your document, then I agree that you wouldn't need to define it first :)